packages/core/src/themes/ds5.ts

```diff
@@ -151,6 +151,26 @@ export const ds5 = mergeTheme(ds5Base, {
         },
       },
     },
+    HvDialogTitle: {
+      classes: {
+        root: {
+          "& .HvStatusIcon-root": {
+            padding: 0,
+          },
+          "& .HvIconContainer-root": {
+            color: `${theme.colors.secondary}!important`,
+          },
+        },
+      },
+    },
+    HvDialogContent: {
+      classes: {
+        root: {
+          borderTop: "none!important",
```

Comment:
we most likely don't need `!important` if we drop the redundant `classes.contentBorder`